symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -342,6 +349,7 @@ public function appendAttachment($file)
                 'file' => $file[$keys[0]],
                 'filename' => is_numeric($keys[0]) ? null : $keys[0],
                 'charset' => null,
+                'cid' => false,
```

Comment:
Same here: I think we don't need to add `cid`.

Reply:
> that doesn't need to be "modernized" anyway.

Why ? Why should we keep 2 syntaxes ?

Reply:
I wouldn't mind removing that legacy stuff. It says "another (un-documented) old format". :-)

https://github.com/symphonycms/symphony-2/blob/3da6eeb9a87096b8ece5f8247830759929f3b107/symphony/lib/toolkit/class.emailgateway.php#L338-L345

Reply:
Good!